data/mods/Magiclysm/items/enchanted_wands.json

```diff
@@ -36,9 +36,16 @@
     "material": [ "wood" ],
     "symbol": "|",
     "color": "brown",
-    "flags": [ "BELT_CLIP", "NONCONDUCTIVE" ],
-    "initial_charges": 20,
-    "max_charges": 20,
+    "flags": [ "BELT_CLIP", "NONCONDUCTIVE", "NO_UNLOAD", "NO_RELOAD" ],
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE",
+        "holster": true,
+        "ammo_restriction": { "crystallized_mana": 20 },
+        "max_contains_volume": "20 L",
```

Comment:
Is it working for non-CONTAINER pockets?

Reply:
i'm not sure what you mean?

Reply:
I mean `max_contains_volume` and `max_contains_weight` probably aren't read for MAGAZINE pockets - https://travis-ci.org/github/CleverRaven/Cataclysm-DDA/jobs/706684568#L1348-L1369

Reply:
yes, you're right but it's not because it's a MAGAZINe pocket, it's because it has an ammo_restriction

Reply:
Oh, right - found the code.